.github/workflows/package.yml

```diff
@@ -17,15 +17,15 @@ jobs:
           lfs: true
       - name: Cache Vulkan SDK
         id: cache-vulkan
-        uses: actions/cache@v2
+        uses: actions/cache@v4
         with:
           path: ${{ env.VULKAN_SDK }}
           key: vulkan-${{ env.VULKAN_VERSION }}
       - name: Install Vulkan SDK
         if: steps.cache-vulkan.outputs.cache-hit != 'true'
         run: |
           Invoke-WebRequest -Uri "https://sdk.lunarg.com/sdk/download/${{ env.VULKAN_VERSION }}/windows/VulkanSDK-${{ env.VULKAN_VERSION }}-Installer.exe" -OutFile vulkan.exe
-          7z x -aoa vulkan.exe "-o$env:VULKAN_SDK" Lib
+          7z x -aoa vulkan.exe "-o$env:VULKAN_SDK" Lib share/vulkan/registry/vk.xml
```

Comment:
Should we just extract everything? Might avoid future annoyance.

Reply:
I'm hopeful that this is just a one-off. The fewer files we extract, the quicker (and less resource-heavy) the GitHub actions will be. I checked on my own machine, and the extracted version is 809 MB if we include everything but only 202 MB if we stick with this. (It would be less we need to cache as well.)

I have a weak preference towards just extracting what we need, but I'm happy to switch to extracting everything if you would prefer.